entrypoint.sh

```diff
@@ -22,6 +24,6 @@ wait_for_couchdb
 
 # Set up system databases
 echo "Setting up CouchDB system databases..."
-docker exec $NAME curl -sS 'http://127.0.0.1:5984/_users' -X PUT -H 'Content-Type: application/json' --data '{"id":"_users","name":"_users"}' > /dev/null
-docker exec $NAME curl -sS 'http://127.0.0.1:5984/_global_changes' -X PUT -H 'Content-Type: application/json' --data '{"id":"_global_changes","name":"_global_changes"}' > /dev/null
-docker exec $NAME curl -sS 'http://127.0.0.1:5984/_replicator' -X PUT -H 'Content-Type: application/json' --data '{"id":"_replicator","name":"_replicator"}' > /dev/null
+docker exec $NAME curl -sS "http://${COUCHDB_USER}:${COUCHDB_PASSWORD}@localhost:5984/_users" -X PUT -H 'Content-Type: application/json' --data '{"id":"_users","name":"_users"}' > /dev/null
```

Comment:
With this step will we avoid the couchdb service? wouldn't be better to use it without username/password as it is in the testing workflow?

Reply:
Couchdb3 doesn't work without username and password, I am doing this for amco use only and we dont have use case for couchdb 1.7 that is the one were the password is optional

Reply:
I guess I could handle more logic to support 1.x, but at this point not sure if would be something we want/care